app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,13 @@
+class LikesController < ApplicationController
+  def create
+    @user = current_user
+    @post = @user.posts.find(params[:post_id])
+    @like = Like.new(user_id: current_user.id, post: @post)
+
+    if @like.save
+      redirect_to user_post_path(params[:user_id], params[:post_id]), notice: 'like successfully saved.'
+    else
+      redirect_to user_post_path(params[:user_id], params[:post_id]), alert: 'like could not be saved.'
+    end
```

Comment:
After creating a post, I tried adding a like, and it didn't work, the likes counter didn't update. I accessed the list of likes from the rails consoleto make sure, and it was empty: 
![image](https://github.com/kkolade/rails_blog_app/assets/93520178/6c470c3e-7e1c-41e8-9604-aaf261a2e455)
Kindly fix this issue.